docs/lib/analytics/menu.json

```diff
@@ -7,6 +7,7 @@
     "identifyuser",
     "streaming",
     "storing",
-    "personalize"
+    "personalize",
+    "escapehatch"
```

Comment:
we duplicate the list of sections from the menu in the getting-started page, so we should add the escapehatch page link at the bottom of the getting started guide as well

Reply:
duplicated